submit-web/src/components/SubmissionItem/InternalDocuments/PendingRow.tsx

```diff
@@ -52,13 +57,29 @@ export default function PendingRow({
     setPendingUpload(true);
   });
 
+  const queryClient = useQueryClient();
+  const accountProject = queryClient.getQueryData<AccountProject>(
+    getAccountProjectQueryOptions(Number(projectId)).queryKey
+  );
+  const projectName = camelCase(accountProject?.project.name ?? "");
+
+  const submissionItem = queryClient.getQueryData<SubmissionItem>(
+    getSubmissionItemQueryOptions({ itemId: Number(submissionItemId) }).queryKey
+  );
+  const folderMap = {
+    [SUBMISSION_ITEM_TYPE.CONSULTATION_RECORD]: S3_FOLDER.CONSULTATION_RECORDS,
+    [SUBMISSION_ITEM_TYPE.MANAGEMENT_PLAN]: S3_FOLDER.MANAGEMENT_PLANS,
+  };
+  const submissionTypeName = submissionItem?.type?.name;
+  const internalStaffSubFolder = folderMap[submissionTypeName as keyof typeof folderMap] || "";
```

Comment:
I would probably use the get method from lodash for null safety here

Reply:
Example:
![image](https://github.com/user-attachments/assets/2eb0cb1c-1d43-437b-b2c6-45cb570b3103)

Reply:
Thanks, updated to use lodash.